media/css/firefox/releasenotes.less

```diff
@@ -447,6 +447,8 @@ article {
     .section-items {
         > li {
             padding-left: 0;
+            word-wrap:break-word;
+            width:50%;
```

Comment:
I think the `word-wrap` property is all we need here? Setting the `width` to 50% pushes text a bit close to the right edge, and results in some overly skinny text on `<li>` elements without an icon - like the "Reference" line on the Android release notes:

<img width="315" alt="screen shot 2015-08-03 at 10 09 01 am" src="https://cloud.githubusercontent.com/assets/317532/9040455/de6e5002-39c7-11e5-8fc5-67772693b1ff.png">